src/app/components/hearing/hearing.component.html

```diff
@@ -12,9 +12,19 @@
 
   <!-- Reporting Restriction Banner -->
   <app-reporting-restriction
-    *ngIf="vm.case.reporting_restriction"
+    *ngIf="vm.case.reporting_restriction && state !== 'OrderFailure'"
     [reportingRestriction]="vm.case.reporting_restriction"
   ></app-reporting-restriction>
+  <ng-container *ngIf="state === 'OrderFailure'">
+    <h1 class="govuk-heading-xl govuk-!-margin-bottom-8">
+      You do not have permission to get audio for this courthouse.
+    </h1>
+    <p class="govuk-body">
+      If you believe you should have permission,
+      <a href="mailto:{{ support?.emailAddress }}" class="govuk-link">contact {{ support?.name }}</a
+      >.
+    </p>
+  </ng-container>
```

Comment:
Could this use `<app-forbidden></app-forbidden>` with the addition of a new `Input` property to override the default heading?